cdap-data-fabric/src/test/java/co/cask/cdap/spi/metadata/MetadataStorageTest.java

```diff
@@ -408,6 +409,39 @@ public void testSearchDescription() throws IOException {
     mds.batch(batch(new Drop(app1), new Drop(app2)));
   }
 
+  @Test
+  public void testSearchOnTTL() throws Exception {
+    MetadataStorage mds = getMetadataStorage();
+
+    final String ns1 = "ns1";
```

Comment:
doesn't seem like these need to be final

Reply:
oh yes that is a copy&paste from a previous test case